src/AccordionItemBody/accordion-item-body.js

```diff
@@ -30,6 +30,19 @@ const AccordionItemBody = ({ id, expanded, children, className, hideBodyClassNam
         },
     );
     const ariaHidden = !expanded;
+    if (role === 'tabpanel') {
+        return (
+            <div
+                id={id}
+                className={bodyClass}
+                aria-hidden={ariaHidden}
+                aria-labelledby={id.replace('accordion__body-', 'accordion__title-')}
```

Comment:
I think this should apply even if `role` isn't `tabpanel` – there still will be a title element for the block, right?

Reply:
Good point